jstp.js

```diff
@@ -13,9 +13,11 @@ jstp.RemoteProxy = require('./lib/remote-proxy');
 jstp.Connection = require('./lib/connection');
 jstp.Server = require('./lib/server');
 
-jstp.tcp = require('./lib/transport.tcp');
-jstp.ipc = require('./lib/transport.ipc');
-jstp.ws = require('./lib/transport.ws');
+jstp.net = require('./lib/net');
+jstp.tls = require('./lib/tls');
+jstp.ws = require('./lib/ws');
+jstp.wss = require('./lib/wss');
+jstp.wsBrowser = require('./lib/ws-browser');
```

Comment:
We should require just `ws` here, resolving it to the actual implementation is up to the module loader (either Node's or webpack's).